nautobot/extras/tests/test_jobs.py

```diff
@@ -425,10 +428,18 @@ def test_job_class_task_queues_override(self, mock_get_celery_queues):
             </select><br><span class="helptext">The task queue to route this job to</span></td></tr>
             <tr><th><label for="id__commit">Commit changes:</label></th>
             <td><input type="checkbox" name="_commit" placeholder="Commit changes" id="id__commit" checked><br>
-            <span class="helptext">Commit changes to the database (uncheck for a dry-run)</span></td></tr>""",
+            <span class="helptext">Commit changes to the database (uncheck for a dry-run)</span>
+            <input type="hidden" name="_profile" value="False" id="id__profile"></td></tr>""",
             form.as_table(),
         )
 
+    def test_job_profiling(self):
+        module = "test_profiling"
+        name = "TestProfilingJob"
+
+        # The job itself contains the 'assert' by loading the resulting profiling file from the workers filesystem
```

Comment:
Since the jobs run by `create_job_result_and_run_job` run locally we should probably just assert that the file exists here in the test and then clean it up.

Reply:
Done!